pkg/db/db.go

```diff
@@ -41,6 +39,15 @@ func WithOCIArtifact(art *oci.Artifact) Option {
 	}
 }
 
+
+// WithDBRepository takes a dbRepository
+func WithDBRepository(dbRepository string) Option {
```

Comment:
The benefit of functional options is that it allows to define default values. The default db repository should be defined here.
https://github.com/aquasecurity/trivy/blob/003173f7f29e5c9767d0f39e941ccf7cb1700d10/pkg/db/db.go#L70